.chloggen/pubsub-ignore-encoding-error.yaml

```diff
@@ -0,0 +1,16 @@
+change_type: enhancement
```

Comment:
breaking if you add a new metric enabled by default.

Reply:
I've changed the release note. Is this stated somewhere that a new metric is a breaking change? I like to read the guidelines to make better release notes, I didn't see this stated in the CONTRIBUTING.md file.